src/code.py

```diff
@@ -501,6 +517,38 @@ def handle_key_press(key):
             if left_tg.contains((mouse_tg.x, mouse_tg.y, 0)):
                 page_left()
 
+    if "screensaver" in launcher_config.data:
+        if last_interaction_time + SCREENSAVER_TIMEOUT < now:
+            if display.auto_refresh:
+                display.auto_refresh = False
+
+            # show the screensaver
+            if screensaver is None:
+                m = __import__(launcher_config.data["screensaver"]["module"])
+                cls = getattr(m, launcher_config.data["screensaver"]["class"])
```

Comment:
When I went through the fish_screensaver.py looking for the class name it occurred to me that the code could look for a member of "m" for which the class name ended with "ScreenSaver" and use that class if one wasn't found in launcher_config. 

I know that means screen saver developers would need to know and follow the standard to be useful but you could still check if one was defined in launcher_config and only use the "ScreenSaver" method if one wasn't defined. I think that makes it easier for the user to setup since all they have to do is look through the screensavers folder for python files that might be screen savers.....